src/main/java/org/spongepowered/api/data/manipulator/mutable/item/MapItemData.java

```diff
@@ -24,14 +24,63 @@
  */
 package org.spongepowered.api.data.manipulator.mutable.item;
 
+import com.flowpowered.math.vector.Vector2i;
 import org.spongepowered.api.data.manipulator.DataManipulator;
 import org.spongepowered.api.data.manipulator.immutable.item.ImmutableMapItemData;
+import org.spongepowered.api.data.value.BoundedValue;
+import org.spongepowered.api.data.value.mutable.MutableBoundedValue;
+import org.spongepowered.api.data.value.mutable.Value;
 import org.spongepowered.api.item.ItemTypes;
 import org.spongepowered.api.item.inventory.ItemStack;
+import org.spongepowered.api.map.MapCanvas;
+import org.spongepowered.api.world.World;
 
 /**
  * Represents an {@link DataManipulator} hosting the specific map
  * information of an {@link ItemStack} of the type {@link ItemTypes#FILLED_MAP}.
+ * Warning: This will affect all maps with this id (damage value).
```

Comment:
What does this warning actually mean? If you mean that a `MapData` instance can be applied/cloned to another map `ItemStack` and the mutations can affect *that other* `ItemStack`, then that's against the Data API contract that we're always returning copies of data, never the live data (unless it's impossible or by structure).